test/integration_test.js

```diff
@@ -99,3 +104,78 @@ function clearConnectorFolder() {
         console.warn(e);
     }
 }
+
+// Start e2e
+chromedriver.start(); // on port 9515
+process.on('exit', chromedriver.stop);
+
+describe('plotly database connector', function Spec() {
+    this.timeout(1000 * 60 * 10);
+
+    const initialize = async () => {
+        // Open the app
+        await delay(1000); // wait chromedriver start time
+        this.driver = new webdriver.Builder()
+        .usingServer('http://localhost:9515')
+        .withCapabilities({
+            chromeOptions: {
+                binary: electronPath,
+                args: [`app=${path.resolve()}`]
+            }
+        })
+        .forBrowser('electron')
+        .build();
+
+        // Initialize credentials
+        clearConnectorFolder();
+        saveCredentials();
+
+        // class helpers
+        this.findEls = (args) => this.driver.findElements(args);
+        this.findEl = (args) => this.driver.findElement(args);
+        this.byClass = (args) => webdriver.By.className(args);
+        this.byId = (args) => webdriver.By.id(args);
+        this.byCss = (args) => webdriver.By.css(args);
+        this.byPath = (args) => webdriver.By.xpath(args);
+        this.fillInput = (id, value) => (
+            this.findEl(this.byId(id))
+        ).sendKeys(value);
+    }
+
+    before(initialize);
+
+    it('should work', async () => {
```

Comment:
@n-riesco Sorry, I meant this test. It was there before (I did not write it)

Reply:
I changed the description, but the test is the same.

Reply:
_sigh_, yeah.. I noticed it now. All good !!

Reply:
I guess you wnat to remove this: